src/test/regress/sql/propagate_statistics.sql

```diff
@@ -0,0 +1,43 @@
+CREATE SCHEMA "statistics'test";
```

Comment:
I miss some tests that would fail if quoting mistakes were made.

This like tables with columns having capital letters, which require proper quotes.

This also needs to be performed twice, once with the statistics created before the table gets distributed and once after.

Same would be for a schemaname that requires quotes, and tablenames that would require quotes.

The different quoted things (schema, columns, tables) can all be in the same test. However we should exercise them on both codepaths for the sql ddl generation.